lib/mjsonwp/errors.js

```diff
@@ -308,6 +308,26 @@ class BadParametersError extends ES6Error {
   }
 }
 
+class ProxyRequestUnsuccessfulError extends ES6Error {
```

Comment:
I don't think "Unsuccessful" is necessary here. An error, by definition, marks something unsuccessful.

I'd rather see `ProxyRequestError`.